src/js/content_scripts/instance.js

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
"instance" is far far too generic.  What is it an "instance" of?  It seems like another input mapper?